application/actions/extrasmware.go

```diff
@@ -0,0 +1,16 @@
+package actions
+
+import (
+	"github.com/gobuffalo/buffalo"
+)
+
+func setHTTPExtras(next buffalo.Handler) buffalo.Handler {
+	return func(c buffalo.Context) error {
+		newExtra(c, "request_method", c.Request().Method)
```

Comment:
I think these are already included in the log, right?

Reply:
I didn't find them in Rollbar when I went looking for them.

Reply:
They're added by `reportError` so if that doesn't get called, because there's no context, for example, they won't be added to extras (probably not sent to rollbar at all). But then again, this middleware won't do any good either, if `reportError` isn't used.